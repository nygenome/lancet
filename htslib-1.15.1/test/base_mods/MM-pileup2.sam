@CO	With added CHeBI codes and mod-code combining.
@SQ	SN:I	LN:999
r1	0	I	1	0	36M	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	DF?GCH88.EG8.7@E9G8A?H9.:C?8,@,,9F@A	Mm:Z:C+m,2,2,1,4,1;C+76792,6,7;	Ml:B:C,128,153,179,204,230,159,6
r2	0	I	1	0	36M	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	*	Mm:Z:C+mh,2,2,0,0,4,1;N+n,15;	Ml:B:C,77,159,103,133,128,108,154,82,179,57,204,31,240
