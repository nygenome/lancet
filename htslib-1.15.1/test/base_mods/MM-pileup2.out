I	0	AA	D~
I	1	GG	F~
I	2	CC	?~
I	3	TT	G~
I	4	CC	C~
I	5	TT	H~
I	6	C[+m128]C[+m77+h159]	8~
I	7	CC	8~
I	8	AA	.~
I	9	GG	E~
I	10	AA	G~
I	11	GG	8~
I	12	TT	.~
I	13	CC	7~
I	14	GG	@~
I	15	NN[+n240]	E~
I	16	AA	9~
I	17	C[+m153]C[+m103+h133]	G~
I	18	GG	8~
I	19	C[+(76792)159]C[+m128+h108]	A~
I	20	C[+m179]C[+m154+h82]	?~
I	21	AA	H~
I	22	TT	9~
I	23	YY	.~
I	24	CC	:~
I	25	GG	C~
I	26	CC	?~
I	27	GG	8~
I	28	CC	,~
I	29	GG	@~
I	30	CC	,~
I	31	C[+m204]C[+m179+h57]	,~
I	32	AA	9~
I	33	CC	F~
I	34	C[+m230+(76792)6]C[+m204+h31]	@~
I	35	AA	A~
